impl/src/main/java/io/opencensus/impl/internal/DisruptorEventQueue.java

```diff
@@ -99,6 +103,8 @@
   // Ring Buffer for the {@link Disruptor} that underlies the queue.
   private final RingBuffer<DisruptorEvent> ringBuffer;
 
+  private DisruptorEnqueuer enqueuer = null;
```

Comment:
This field should be volatile, since it could be accessed by multiple threads.

Reply:
ah, missed this one. fixed.